Hunted Mobile/Hunted Mobile/Hunted Mobile/ViewModel/MainPageViewModel.cs

```diff
@@ -81,9 +83,11 @@ public class MainPageViewModel : BaseViewModel {
             SubmitButtonIsEnable = false;
             await GetAll();
 
+            Game toBeJoined = await _gameRepository.GetGame(InviteKeyModel.GameId);
+
             // Navigate when InviteKey is valid
-            if(IsValid = ValidationHelper.IsFormValid(InviteKeyModel, _page) && !IsOverlayVisible) { 
-                await NavigateToEnterUsernamePage();
+            if((IsValid = ValidationHelper.IsFormValid(InviteKeyModel, _page) && !IsOverlayVisible) || toBeJoined.Status == "on-going" || toBeJoined.Status == "paused" || toBeJoined.Status == "finished") { 
+                await NavigateToEnterUsernamePage(toBeJoined.Status);
```

Comment:
Van deze string waardes zoals `paused` zou ik ergens misschien wel statisch plaatsen, zodat je deze waardes maar één keer hoef te plaatsen. Als er dan een typo zou zijn of de statussen worden hernoemd, dan hoef je dit in één bestand aan te passen. 
Deze manier heb ik weleens gebruikt: https://stackoverflow.com/questions/8588384/how-to-define-an-enum-with-string-value#answer-8588424
Dan maak ik gewoon een statische klasse met statische waardes